README.md

```diff
@@ -353,6 +353,17 @@ which you can provide as `baseIRI: string` to the config.
 | dT             | ✓ | ✓ | ? |   |   |
 | bool           | ✓ | ✓ | ? |   |   |
 
+### SPARQL 1.2
+
+Sparqlee looks forward and already implements some SPARQL 1.2 specification functions.
+
+Currently, this is restricted to the [extended date](https://github.com/w3c/sparql-12/blob/main/SEP/SEP-0002/sep-0002.md) functionality.
+Please note that the new build-in `ADJUST` function has not been implemented due to package dependencies.
+
+For the extended date functionality, an additional context item has been added, `implicitTimezone`.
+The choice was made to default to UTC time so the `implicitTimezone` does not change over time
+(soo it is not daylight saving time dependent).
```

Comment:
```suggestion
(i.e., it is not dependent on daylight saving time).
```

Reply:
Ah, seems like this is outdated too, we altered this to default to the timezone in `now`.